index.bs

```diff
@@ -290,41 +283,50 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
+         then return true.
+
+      1. Otherwise, [=set/for each=] |item| in the <a>allowlist</a>:
+
+        1. If |item| is [=same origin-domain=] with <var>origin</var>, then
+           return true.
+
+      1. Return false.
+    </div>
   </section>
   <section>
     <h3 id="default-allowlists">Default Allowlists</h3>
     <p>Every <a>policy-controlled feature</a> has a <dfn export
     lt="default allowlist|default allowlists"
     for="policy-controlled feature">default allowlist</dfn>. The <a>default
-    allowlist</a> determines whether the feature is allowed in a document with
-    no declared policy in a [=/top-level traversable=], and also whether access
-    to the feature is automatically delegated to documents in [=child
-    navigables=].</p>
+    allowlist</a> determines whether the feature is allowed in a {{Document}} with
+    no <a for="permissions policy">declared policy</a> in a [=/top-level
+    traversable=], and also whether access to the feature is automatically
+    delegated to documents in [=child navigables=].</p>
     <p>The <a>default allowlist</a> for a <a
     data-lt="policy-controlled feature">feature</a> is one of these values:</p>
     <dl>
       <dt><dfn for="default allowlist" export><code>*</code></dfn></dt>
-      <dd>The feature is allowed in documents in top-level traversables by
-      default, and when allowed, is allowed by default to documents in child
-      navigables.</dd>
+      <dd>The feature is allowed in {{Document}}s in [=/top-level
+      traversables=] by default, as well as those in all [=child navigables=].
+      It can be disallowed in [=child navigables=] by explicitly supplying a
+      [=container policy=] on the [=navigable container=] that overrides this
+      default (or in any [=/navigable=], by delivering the {{Document}} with a
+      suitable <a http-header>`Permissions-Policy`</a> header).</dd>
       <dt><dfn for="default allowlist" export><code>'self'</code></dfn></dt>
-      <dd>The feature is allowed in documents in top-level traversables by
-      default, and when allowed, is allowed by default to same-origin domain
-      documents in child navigables, but is disallowed by default in
-      cross-origin documents in child navigables.</dd>
+      <dd>The feature is allowed in [=navigable/active document|documents=] in
+      [=/top-level traversables=] by default, as well as those in [=child
+      navigables=] whose [=navigable/active document|document=] is
+      [=same origin=] with its [=navigable/parent=]'s [=navigable/active
+      document|document=], when allowed in that {{Document}}. It is disallowed
+      by default in [=child navigables=] whose [=navigable/active
+      document|document=] is cross-origin with its [=navigable/parent=]'s
+      [=navigable/active document|document=], but can be enabled by explicitly
+      supplying a [=container policy=] that delegates the feature.</dd>
```

Comment:
@clelland Given some of the discussion in https://github.com/w3c/webappsec-permissions-policy/pull/511#discussion_r1187708674 and https://github.com/w3c/webappsec-permissions-policy/pull/511#discussion_r1199294407, let me know if you think I should just remove the sentence from lines 325-329, since technically other conditions have to be satisfied _besides_ a lenient container policy.

Reply:
Maybe delete everything from "but can be", since the previous part of the statement is true. The bit about "can be enabled with a container policy" is the part that might also require other conditions. Does that make sense?

Reply:
Yep, definitely. The conditions seem too many to enumerate here, so paring this down makes sense. Done.